index.js

```diff
@@ -193,7 +285,11 @@ async function main(options) {
       },
     });
     try {
-      await doAction(chatApi, page, task, options);
+      if (options.autogpt) {
+        await doAutoGPTAction(chatApi, task);
```

Comment:
```suggestion
        await doAutoGPTAction(page, chatApi, task, option);
```

you can pass `page` and `option` in here too.